docs/tests/build.rst

```diff
@@ -103,13 +105,38 @@ When downloading source, we by default use the last of the url path as
 the filename, or a hash of the url if is no suitable name. This
 parameter to overrides the default behavior with a pre-defined filename.
 
+create\_files
+^^^^^^^^^^^^^
+
+This build attribute lets you create files relative to the build directory
+at build time. Note that any existing files with conflicting names will be
+overwritten.
+
+.. code-block:: yaml
+
+    mytest:
+      build:
+        source_location: mytest.zip
+        cmds: 'make'
+        make_files:
+          # Files created this way will be actual files, not symlinks.
```

Comment:
That's not true. These still go into the build directory during the build. When the build is 'copied' to the test_run, they'll end up being symlinks.